2021-04-25 17:44:53,890 INFO    MainThread:57114 [wandb_setup.py:_flush():69] setting env: {}
2021-04-25 17:44:53,890 INFO    MainThread:57114 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-25 17:44:53,890 INFO    MainThread:57114 [wandb_init.py:_log_setup():336] Logging user logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_174453-8zcc2ane/logs/debug.log
2021-04-25 17:44:53,890 INFO    MainThread:57114 [wandb_init.py:_log_setup():337] Logging internal logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_174453-8zcc2ane/logs/debug-internal.log
2021-04-25 17:44:53,891 INFO    MainThread:57114 [wandb_init.py:init():369] calling init triggers
2021-04-25 17:44:53,891 INFO    MainThread:57114 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-25 17:44:53,891 INFO    MainThread:57114 [wandb_init.py:init():418] starting backend
2021-04-25 17:44:53,892 INFO    MainThread:57114 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-25 17:44:53,907 INFO    MainThread:57114 [backend.py:ensure_launched():123] starting backend process...
2021-04-25 17:44:53,915 INFO    MainThread:57114 [backend.py:ensure_launched():128] started backend process with pid: 57130
2021-04-25 17:44:53,930 INFO    MainThread:57114 [wandb_init.py:init():423] backend started and connected
2021-04-25 17:44:53,932 INFO    MainThread:57114 [wandb_init.py:init():463] updated telemetry
2021-04-25 17:44:53,933 INFO    MainThread:57114 [wandb_init.py:init():482] communicating current version
2021-04-25 17:44:54,617 INFO    MainThread:57114 [wandb_init.py:init():487] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-25 17:44:54,617 INFO    MainThread:57114 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-25 17:44:54,799 INFO    MainThread:57114 [wandb_init.py:init():520] starting run threads in backend
2021-04-25 17:44:54,885 INFO    MainThread:57114 [wandb_run.py:_console_start():1524] atexit reg
2021-04-25 17:44:54,886 INFO    MainThread:57114 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-25 17:44:54,886 INFO    MainThread:57114 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-25 17:44:54,889 INFO    MainThread:57114 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-25 17:44:54,889 INFO    MainThread:57114 [wandb_init.py:init():544] run started, returning control to user process
2021-04-25 17:45:15,691 INFO    MainThread:57114 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-25 17:45:15,692 INFO    MainThread:57114 [wandb_run.py:_restore():1466] restore
2021-04-25 17:45:17,863 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 17:45:18,229 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 17:45:18,820 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 1677
}

2021-04-25 17:45:19,017 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 2025
}

2021-04-25 17:45:19,119 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2025
  total_bytes: 2025
}

2021-04-25 17:45:19,222 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2025
  total_bytes: 2025
}

2021-04-25 17:45:19,324 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2025
  total_bytes: 2025
}

2021-04-25 17:45:19,426 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2025
  total_bytes: 2025
}

2021-04-25 17:45:19,529 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2025
  total_bytes: 2025
}

2021-04-25 17:45:19,632 INFO    MainThread:57114 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2025
  total_bytes: 2025
}

2021-04-25 17:45:20,789 INFO    MainThread:57114 [wandb_run.py:_show_files():1838] logging synced files
